internal/services/kusto/kusto_cluster_resource_test.go

```diff
@@ -972,6 +985,120 @@ resource "azurerm_kusto_cluster" "test" {
 `, data.RandomInteger, data.Locations.Primary, data.RandomString, data.RandomString, data.RandomString, data.RandomString, data.RandomString, data.RandomString, data.RandomString)
 }
 
+func (KustoClusterResource) vnetUpdate(data acceptance.TestData) string {
```

Comment:
Maybe allow the `public_network_access_enabled` value to be passed in as a parameter for code reuse here?

```suggestion
func (KustoClusterResource) vnetUpdate(data acceptance.TestData, publicNetworkAccessEnabled bool) string {
```